### ISSUE TITLE
Infinite Loop occurs when I try to execute implementation's command in a shell

### ISSUE THREAD

**Issue by mjhd-devlion** [KeenS/CIM#59]
Here's the steps to reproduce this problem:
```bash
$ cim use clisp-system
$ sbcl  (another implementation)
(inifite loop...)
```
This problem also occurs any implementations (and versions) pair.  

I tried to fix this problem,
and I found my config file (`current.$CIM_ID`) being like this:
```
LISP_IMPL=clisp-system
abcl=abcl
alisp=alisp
ccl=ccl
clisp=clisp-system
ecl=ecl
lisp=lisp
sbcl=sbcl
```
I think it should be like this:
```
LISP_IMPL=clisp-system
abcl=abcl-system
alisp=alisp-system
ccl=ccl-system
clisp=clisp-system
ecl=ecl-system
lisp=lisp-system
sbcl=sbcl-system
```
So I rewrote the file, but the `cim use ***` command resets my config file.

Finally, I edited `.cim/scripts/cim_cmd_use` file's line 30 to be `echo "${var%-*}=$var-system"` temporarily.
https://github.com/KeenS/CIM/blob/master/scripts/cim_cmd_use#L30

**Comment by fukamachi** [comment]
Sounds like the same problem as #35.

**Comment by KeenS** [comment]
Hmm... I thought I eliminated this bug while ago but it occurred. I'll fix it later.
BTW consider migrating to [roswell](https://github.com/snmsts/roswell) if it satisfies you; it is stable.

**Comment by mjhd-devlion** [comment]
Thank you!

I know roswell is powerful, but I like CIM's pretty REPL.
I'll use both for now. Thank you.

**Comment by KeenS** [comment]
@mjhd-devlion I found this occurs only on OS X. Because I'm a Ubuntu user, it'll take some time to fix this bug. In develop c252c33fd8b5a23c3f1020b155118b7d0e7d65b8 I think this bug is fixed but I'm not sure.
If you don't mind, try

    $ CIM_INSTALL_BRANCH=develop cim get

.

**Comment by mjhd-devlion** [comment]
@KeenS I've confirmed the problem was fixed to apply [c252c33](d8b5a23c3f1020b155118b7d0e7d65b8). Thank you!

**Comment by KeenS** [comment]
I'm glad to hear fix report.
